ui/packages/catalog-ui-search/src/main/webapp/component/result-form/result-form.collection.js

```diff
@@ -49,6 +50,21 @@ module.exports = Backbone.AssociatedModel.extend({
       this.addResultForms()
       this.doneLoading()
     })
+    let self = this
```

Comment:
✏️ think you could also define the `onMessage` function here and pass it in to avoid this assignment